grpc/interceptor/logger.go

```diff
@@ -19,22 +20,33 @@ func Logger(l *logrus.Logger) grpc.UnaryServerInterceptor {
 	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
 		ctx = addLoggerToContext(l, ctx)
 		entry := logger.FromContext(ctx)
-		name := info.FullMethod
+
+		// regex to change /service.Service/Endpt -> Endpt
+		r := regexp.MustCompile(`\/([A-Za-z])\w+\.([A-Za-z])\w+\/`)
```

Comment:
@ehudc What do you think if you extract this as a const `FormatServiceEndPoint` ?